controllers/clusteroperator_test.go

```diff
@@ -5,15 +5,15 @@ import (
 	"testing"
 	"time"
 
+	osconfigv1 "github.com/openshift/api/config/v1"
```

Comment:
Why was this moved? Usually we're keeping the imports in the following format:

std libs

outside libs

openshift libs

Reply:
aaah, OK so goimports does
```
std libs

outside libs

local libs
```
I can adjust the definition of "local" to github.com/openshift

Reply:
aaah, OK so goimports does
```
std libs

outside libs

local libs
```
I can adjust the definition of "local" to github.com/openshift